packages/react-native/ReactCommon/react/renderer/components/view/platform/macos/react/renderer/components/view/HostPlatformViewProps.cpp

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include "HostPlatformViewProps.h"
+
+#include <react/featureflags/ReactNativeFeatureFlags.h>
+#include <react/renderer/components/view/conversions.h>
+#include <react/renderer/core/graphicsConversions.h>
+#include <react/renderer/core/propsConversions.h>
+
+namespace facebook::react {
+
+HostPlatformViewProps::HostPlatformViewProps(
+    const PropsParserContext& context,
+    const HostPlatformViewProps& sourceProps,
+    const RawProps& rawProps)
+    : BaseViewProps(context, sourceProps, rawProps),
+      hostPlatformEvents(
+          ReactNativeFeatureFlags::enableCppPropsIteratorSetter()
+              ? sourceProps.hostPlatformEvents
+              : convertRawProp(
+                    context,
+                    rawProps,
+                    sourceProps.hostPlatformEvents,
+                    {})),
+      enableFocusRing(
+          ReactNativeFeatureFlags::enableCppPropsIteratorSetter()
+              ? sourceProps.enableFocusRing
+              : convertRawProp(
+                    context,
+                    rawProps,
+                    "enableFocusRing",
+                    sourceProps.enableFocusRing,
+                    true)),
+      focusable(
+          ReactNativeFeatureFlags::enableCppPropsIteratorSetter()
+              ? sourceProps.focusable
+              : convertRawProp(
+                    context,
+                    rawProps,
+                    "focusable",
+                    sourceProps.focusable,
+                    {})),
+      draggedTypes(
+          ReactNativeFeatureFlags::enableCppPropsIteratorSetter()
+              ? sourceProps.draggedTypes
+              : convertRawProp(
+                    context,
+                    rawProps,
+                    "draggedTypes",
+                    sourceProps.draggedTypes,
+                    {})),
+      tooltip(
+          ReactNativeFeatureFlags::enableCppPropsIteratorSetter()
+              ? sourceProps.tooltip
+              : convertRawProp(
+                    context,
+                    rawProps,
+                    "tooltip",
+                    sourceProps.tooltip,
+                    {})),
+      validKeysDown(
```

Comment:
Replaced completely with `keyDownEvents` since https://github.com/microsoft/react-native-macos/pull/1615 . Can we update the props here?

Reply:
Talked offline, this doesn't target main, so this is fine for now.